nodezator/fontsman/cache.py

```diff
@@ -161,6 +174,8 @@ def get_font(font_path, desired_height):
         ### surface of an arbitrary character (space) when
         ### rendered
 
+        raise_if_unreachable(desired_height, size)
```

Comment:
note: infinite loop was possible for `desired_height = 1`, adding `raise` avoids this